webapi/src/it/scala/org/knora/webapi/e2e/v1/KnoraSipiScriptsV1ITSpec.scala

```diff
@@ -109,13 +109,22 @@ class KnoraSipiScriptsV1ITSpec extends ITKnoraFakeSpec(KnoraSipiScriptsV1ITSpec.
             // get the filename
             val filename = sipiPostResponseJson.fields("filename").asInstanceOf[JsString].value
 
-            // Send a GET request to Sipi, asking for the preview image
-            val sipiGetRequest01 = Get(previewPath)
-            val sipiGetResponseJson01 = getResponseString(sipiGetRequest01)
+            /**
+              * Send a GET request to Sipi, asking for the preview image.
+              * With testcontainers it is not possible to know the random port
+              * in advance, so the started sipi container cannot be configured
+              * correctly.
```

Comment:
I don't understand. Does this mean that this test can't work with testcontainers? Does something need to be fixed here?

Reply:
I have a workaround for this. In the tests, I rewrite the returned standard URL from Sipi to the correct one. We are probably overspecifying things in Sipi, which makes the whole thing a bit unflexible.

Reply:
Could you clarify the comment? "the started sipi container cannot be configured correctly" makes it seem as if something is broken and needs to be fixed.

Reply:
done